web/modules/custom/ai_screening_project/src/Helper/ProjectHelper.php

```diff
@@ -261,6 +268,19 @@ public function isCorrupted(EntityInterface $entity): bool {
     return FALSE;
   }
 
+  /**
+   * Decide if project is finished.
+   */
+  public function isFinished(EntityInterface $entity): bool {
+    if ($this->isProject($entity) && $entity instanceof FieldableEntityInterface) {
+      if ($entity->hasField(self::FIELD_STATE)) {
+        return $entity->get(self::FIELD_STATE)->getString() === 'finished';
```

Comment:
Should we use an enum for the state values (similar to https://github.com/itk-dev/ai-screening/blob/develop/web/modules/custom/ai_screening_project_track/src/Status.php)?